src/main/scala/org/constellation/p2p/PeerAPI.scala

```diff
@@ -277,14 +279,16 @@ class PeerAPI(override val ipManager: IPManager[IO])(
   }
 
   def routes(address: InetSocketAddress): Route = withTimer("peer-api") {
+    withSimulateTimeout(dao.simulateEndpointTimeout)(ConstellationExecutionContext.edge) {
 //    val id = ipLookup(address) causes circular dependencies and cluster with 6 nodes unable to start due to timeouts. Consider reopen #391
-    // TODO: pass id down and use it if needed
+      // TODO: pass id down and use it if needed
 
-    decodeRequest {
-      encodeResponse {
-        // rejectBannedIP {
-        signEndpoints ~ commonEndpoints ~ enforceKnownIP(address) {
-          getEndpoints(address) ~ postEndpoints ~ mixedEndpoints ~ blockBuildingRoundRoute
+      decodeRequest {
+        encodeResponse {
+          // rejectBannedIP {
+          signEndpoints ~ commonEndpoints ~ enforceKnownIP(address) {
```

Comment:
Please remember that registration endpoints shouldn't have timeouting feature (`signEndpoints`)